project.md

```
@@ -0,0 +1,138 @@
+We use bun as package manager for node and uv as package manager for python.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)